topsort-api-v2.yml

```diff
@@ -566,6 +573,7 @@ components:
           minLength: 1
     EventsRequest:
       type: object
+      additionalProperties: false
```

Comment:
Are we going to enforce this? One thing to have in mind, is that field can never be deleted when this option is used. Otherwise clients may break.

Reply:
I've added a line to enforce this in the events v2 PR; the main reason being that in the past we've found that some marketplaces have had typos in optional fields (e.g. `occurredAt` in events v1) and no one noticed for some time (until Joaquín saw some of the events manually).

That's a fair tradeoff: we can only deprecate fields but never really delete them unless we release a new API version.

Reply:
One alternative I've thought about is that we can be more strict in the sandbox environment and more lax in the production environment so that at least during development marketplaces have better tooling to detect these issues.